packaging/linux/post_install.sh

```diff
@@ -173,6 +173,14 @@ fix_bad_config_perms() {
 
 fix_bad_config_perms
 
+# Associate Keybase with .saltpack files.
+if command -v xdg-mime &> /dev/null ; then
```

Comment:
Not everyone runs post install. Instead of editing postinstall, how about putting the xml file directly in /usr/share/mime/application in `build_binaries.sh`? And adding application/x-saltpack to `keybase.desktop`?

Reply:
That sounds fine, but I couldn't tell whether installing the xml file does more than just copy it.. I think there's an index or two that it rebuilds too. I'll try stracing it.

Reply:
I think @eli-schwartz pointed out `update-mime-database` below which sounds like a good idea.